README.md

```diff
@@ -52,7 +61,21 @@ To run this module, you need the following:
     terraform output -json
     ```
 
-7. Grant access on the enrollment account as described in the [meshcloud public docs](https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform-azure-manually.html#set-up-subscription-provisioning).
+#### If Using an Enterprise Agreement
+1. Grant access on the enrollment account as described in the section [Use an Enteprise Enrollment](https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform-azure-manually.html#use-an-enterprise-enrollment).
+
+#### If Using Microsoft Customer Agreement
+1. Switch to the Tenant Directory that contains your Billing Account and follow the steps to [Register an Application](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#register-an-application) and [Add Credentials](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#add-credentials). Make sure to copy down the **Directory (tenant) ID**, **Application (client) ID**, **Object ID** and the **App Secret** value that was generated. The App Secret is only visible during the creation process.
```

Comment:
It's possible to create app registrations via terraform. We do this for example in the replicator module [here](https://github.com/meshcloud/terraform-azure-meshplatform/blob/3fe3f1be1e65fd5d35c24e8c05eb19f8c6dbbecf/modules/meshcloud-replicator-service-principal/module.tf#L68).

We could have an input `subscription_provisioning_type`. If you choose MCA you have to set a source tenant id and it'll create the enterprise application. It'll then have the necessary information in the output.

Not necessarily needed right away tough.